packages/core/src/components/Table/TableRow/TableRow.module.scss

```diff
@@ -4,11 +4,17 @@
   height: var(--table-row-size);
   display: grid;
   grid-template-columns: var(--table-grid-template-columns);
+  min-width: 100%;
+  width: fit-content;
 
   &[aria-selected="true"] > * {
     background-color: var(--primary-selected-color);
   }
 
+  > * {
+    background-color: var(--primary-background-color);
```

Comment:
Isn't it dangerous? What about components used in rows?

Reply:
It does not affect deeper nested elements beyond the first level, because of the `>`

Reply:
and the first level should always be the table cell element

Reply:
note for myself: target table cell only anyway, for performance concerns